Testing/UnitTests/xASL_ut_master_xASL_ExploreASL.m

```diff
@@ -530,6 +530,7 @@
 % Create dataPar.json
 dataParStruct.x.settings.Quality = 0;
 dataParStruct.x.S.Atlases = {'TotalGM','DeepWM','Hammers','HOcort_CONN','HOsub_CONN','Mindboggle_OASIS_DKT31_CMA'};
```

Comment:
Should we also unit test WB and a bit more WM?

Reply:
WB added.